common/etl/etl.go

```diff
@@ -269,8 +275,22 @@ func readElementFromDisk(decoder Decoder) ([]byte, []byte, error) {
 type bucketState struct {
 	getter ethdb.Getter
 	bucket []byte
+	quit chan struct{}
 }
 
 func (s *bucketState) Get(key []byte) ([]byte, error) {
 	return s.getter.Get(s.bucket, key)
 }
+
+
+func (s *bucketState) Stopped() error {
```

Comment:
I would just use `bool` as a return value there, but otherwise looks good.

Reply:
or, maybe we can have a global ErrStopped so we handle that same shutdown the same way?

Reply:
I think we can extract this into a small common packsge